### DISCUSSION TITLE
Bug report: getListingsByShop endpoint is sending expired listings even when only inactive listings are requested

### DISCUSSION THREAD

**Discussion by KushGoyal** [etsy/open-api#1011] [Q&A]
Bug:

`getListingsByShop` endpoint has a query param to filter results based on the state of the listing. I am requesting `inactive` listings. In the results there are listings which are expired because the `ending_timestamp` is in the past. So such listings should been correctly marked as `expired`.

How to reproduce:

Make a listing `inactive` and then let it expire. The listing will remain in the `inactive` state.

I request the Etsy development team to fix this.

**Comment by Gareth064** [comment]
Hey @KushGoyal 
You will need to email this over to the Etsy Developer team (developer@etsy.com). This GitHub Discussion board is for the community to discuss things and ask for advice from each other. The Etsy Devs don't use for bug/issue tracking etc.

If you do email them, and they give you a response as to why this is happening, then please report back here with what they said so we can all learn from it.
